src/libraries/Microsoft.PowerFx.Core/Public/Values/FormulaValueNew.cs

```diff
@@ -122,6 +124,11 @@ public static ErrorValue NewError(ExpressionError error, FormulaType type)
             return new ErrorValue(IRContext.NotInSource(type), error);
         }
 
+        public static ErrorValue NewError(IEnumerable<ExpressionError> error, FormulaType type)
```

Comment:
Can you add a unit test for code coverage?

Reply:
Added